resources/views/backend/users/create.blade.php

```diff
@@ -3,16 +3,15 @@
 @section('title', __('CREATE USER'))
 
 @section('content')
- <div class="content-wrapper">
-      <h1 class="title_page text-success">
-        @include('flash::message')
-        {{ __('Create user') }}
-      </h1>
+  <div class="content-wrapper">
+    <h1 class="title_page text-success">
+      @include('flash::message')
+      {{ __('Create user') }}
+    </h1>
```

Comment:
You must review again end tag of `<div class="content-wrapper">`, or end tag of `<section class="content">`.
I check some page, it's missed.